### PR TITLE
Added x logo to sprite.svg

### PR DESCRIPTION
**Pull request by nick-mon1**
## Summary

This PR will update the twitter text and logo to match the new X format.

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/nl-update-x-logo/)

### Solution

Updated the sprite command in the gulp file along with the styles.js file and also added the x.svg to the sprite.svg file so that it could then render on the page

> [!NOTE]
> Gulp build commands need to be updated, they are currently written but not hooked into the build commands
> Created a card in trello to capture this work

### How To Test

1. Visit [homepage](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/nl-update-x-logo/) and scroll down to the bottom and check that the twitter is now X and link takes user to correct page. 
2. Visit [authors page](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-update-x-logo/authors/18f-team/)
3. Visit [news or events page](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-update-x-logo/event/2023/09/21/uswds-monthly-call-september-2023/) and look at the share tools sidebar

---

### Dev Checklist

- [x] PR has correct labels
- [x] A11y testing (voice over testing, meets WCAG, run axe tools)
- [x] Code is formatted properly

tagging supervisor for reference - @rayestrada

### CODE REVIEW

**Review by clmedders — APPROVED** [review]

**Review by clmedders — APPROVED** [review]

**Review by mejiaj — CHANGES_REQUESTED** [review]
URL on events seems broken - [Preview →](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-update-x-logo/event/2023/09/21/uswds-monthly-call-september-2023/).

---

Authors also has some odd formatting and still uses word "twitter" on hover.

```
18F Teamon Twitter
```

**Screenshot**
image

[Preview →](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-update-x-logo/authors/18f-team/)

**Review by clmedders — APPROVED** [review]

**Comment by nick-mon1** [comment]
Reviewed @clmedders updates and confirmed all issues in @mejiaj previous comment are now fixed.

File: themes/digital.gov/layouts/authors/list.html [hunk 1]
```diff
@@ -93,9 +93,9 @@ <h1>{{- .Params.display_name | markdownify -}}</h1>
                     {{- if .Params.twitter -}}
                       <li>
                         <a
-                          class="twitter"
+                          class="x"
                           rel="external"
-                          title="{{- .Params.display_name -}} on Twitter"
+                          title="{{- .Params.display_name }} on X"
                           href="http://www.twitter.com/{{- .Params.twitter -}}/"
```
Comment (mejiaj) [inline themes/digital.gov/layouts/authors/list.html:99]: @clmedders @nick-mon1 confirming, we're leaving the domain in `href` the same; i.e. `http://www.twitter.com/`? Either way we should use `https` for security.

File: themes/digital.gov/layouts/partials/core/footer.html [hunk 2]
```diff
@@ -77,13 +77,13 @@ <h4 class="usa-footer__primary-link">Social</h4>
                     focusable="false"
                   >
                     <use
-                      xlink:href="{{ "uswds/img/sprite.svg#twitter" | relURL }}"
+                      xlink:href="{{ "uswds/img/sprite.svg#x" | relURL }}"
                     ></use>
                   </svg>
                   <a
                     class="text-ink hover:text-ink text-no-underline hover:text-underline"
                     href="https://twitter.com/digital_gov"
```
Comment (mejiaj) [inline themes/digital.gov/layouts/partials/core/footer.html:85]: Same here, `twitter.com`. This one already uses `https`.

**Review by mejiaj — COMMENTED** [review]
~Just a question on the domain.~ Per internal discussion; this will stay the same for now.

File: themes/digital.gov/layouts/authors/list.html (hunk 1 above)
Comment (clmedders) [reply themes/digital.gov/layouts/authors/list.html:99]: updated to https:

File: themes/digital.gov/layouts/partials/core/footer.html (hunk 2 above)
Comment (clmedders) [reply themes/digital.gov/layouts/partials/core/footer.html:85]: twitter.com is till the current domain

**Review by mejiaj — DISMISSED** [review]

**Review by clmedders — APPROVED** [review]

**Review by clmedders — APPROVED** [review]

**Pull request merged**